lib/static_data/events.json

```diff
@@ -0,0 +1,326 @@
+[
+  {
+    "id": "333",
```

Comment:
prod database has a `cellName` column, but the JSoN does not - a deliberate exclusion?

Reply:
Yes, `cellName` is being dropped in a migration that is part of this PR